### PR TITLE
doc: update changelog doc

### PR DESCRIPTION
**Pull request by forcodedancing**
### Description
Change log document is not maintained, users/developers should goto BNB Chain Blog or other channel to watch the latest updates. 

### Example
No

### Changes
No

### CODE REVIEW

**Comment by Zfvtgds** [comment]
what is happening ?

File: docs/beaconchain/changelog.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 ========
 
+##  Latest Changes
+
```
Comment (guagualvcha) [inline docs/beaconchain/changelog.md:5]: Shall we just delete this page?

Comment (forcodedancing) [reply docs/beaconchain/changelog.md:5]: sure, agree.

**Comment by forcodedancing** [comment]
> what is happening ?

This doc does catch up to the latest changes. Twitter/Discord/Blog are better places to follow for this.

**Review by guagualvcha — APPROVED** [review]

**Pull request merged**